### PR TITLE
pythonPackages: websocket_client -> websocket-client

### PR DESCRIPTION
**Pull request by SuperSandro2000**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by dotlambda — COMMENTED** [review]
Shouldn't we add an alias?
EDIT: Nvm, you kinda added one.

File: pkgs/top-level/python-aliases.nix [hunk 1]
```diff
@@ -52,4 +52,5 @@ mapAliases ({
   smart_open = smart-open; # added 2021-03-14
   smmap2 = throw "smmap2 has been deprecated, use smmap instead."; # added 2020-03-14
   topydo = throw "python3Packages.topydo was moved to topydo"; # 2017-09-22
+  websocket-client = websocket-client;
```
Comment (dotlambda) [inline pkgs/top-level/python-aliases.nix:55]:
```suggestion
  websocket_client = websocket-client;
```

**Review by dotlambda — CHANGES_REQUESTED** [review]

File: pkgs/top-level/python-aliases.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/top-level/python-aliases.nix]: Good catch. That was not supposed to happen.

File: pkgs/tools/admin/azure-cli/python-packages.nix [hunk 2]
```diff
@@ -499,11 +499,11 @@ let
         };
       });
 
-      websocket_client = super.websocket_client.overridePythonAttrs(oldAttrs: rec {
+      websocket-client = super.websocket-client.overridePythonAttrs(oldAttrs: rec {
         version = "0.56.0";
 
         src = super.fetchPypi {
```
Comment (dotlambda) [inline pkgs/tools/admin/azure-cli/python-packages.nix:505]:
```suggestion
        src = oldAttrs.src.override {
```

File: pkgs/tools/admin/azure-cli/python-packages.nix [hunk 3]
```diff
@@ -499,11 +499,11 @@ let
...
-          inherit (oldAttrs) pname;
+          pname = "websocket_client";
```
Comment (dotlambda) [inline pkgs/tools/admin/azure-cli/python-packages.nix:506]:
```suggestion
```

**Review by dotlambda — CHANGES_REQUESTED** [review]

**Pull request merged**